k-distribution/tests/regression-new/set-symbolic-tests/diff-11-spec.k.out

```diff
@@ -1,5 +1,6 @@
   <k>
-    diffResult ( SetItem ( Y:MyId ) -Set SetItem ( X:MyId ) ) ~> .
+    diffResult ( SetItem ( Y:MyId ) -Set ( S
+    SetItem ( X:MyId ) ) ) ~> .
```

Comment:
This test actually failed before, the wrong output was stored as the expected one. Now passing :heavy_check_mark: